src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -729,8 +730,9 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
         // cache the container
         $dumper = new PhpDumper($container);
 
-        if (class_exists('ProxyManager\Configuration') && class_exists('Symfony\Bridge\ProxyManager\LazyProxy\PhpDumper\ProxyDumper')) {
-            $dumper->setProxyDumper(new ProxyDumper());
+        $proxyDumper = $this->getProxyDumper();
+        if (null !== $proxyDumper) {
```

Comment:
can be put on one line:
`if (null !== $proxyDumper = $this->getProxyDumper()) {`